app/uk/gov/hmrc/tradergoodsprofiles/models/responses/CreateOrUpdateRecordResponse.scala

```diff
@@ -74,18 +72,16 @@ object CreateRecordResponse {
         (json \ "toReview").as[Boolean],
         (json \ "reviewReason").asOpt[String],
         (json \ "declarable").as[String],
-        (json \ "ukimsNumber").as[String],
-        (json \ "nirmsNumber").as[String],
-        (json \ "niphlNumber").as[String],
-        (json \ "locked").as[Boolean],
-        (json \ "srcSystemName").as[String],
```

Comment:
do we not need these fields?

Reply:
`locked` and `srcSystemName` are not part of the spec response for create and update. They are only returned as part of get records

Reply:
`locked` and `srcSystemName` are not part of the spec response for create and update. They are only returned as part of get records